t/cobrand/councils.t

```diff
@@ -11,4 +22,31 @@ foreach my $council (qw/oxfordshire bromley/) {
     };
 }
 
+
+foreach my $test (
+    { cobrand => 'fixmystreet', social => 1 },
+    { cobrand => 'bromley', social => 0 },
+) {
+
+    FixMyStreet::override_config {
+        ALLOWED_COBRANDS => [ { $test->{cobrand} => '.' } ],
```

Comment:
This is fine, but note that it could be `ALLOWED_COBRANDS => [ $test->{cobrand} ]` :)